bigtable-client-core-parent/bigtable-hbase/src/main/resources/com/google/cloud/bigtable/hbase/bigtable-hbase-version.properties

```diff
@@ -0,0 +1 @@
+bigtable-hbase.version=${project.version}
```

Comment:
Please add comment to this file directing people to check com.google.cloud.bigtable.hbase.BigtableHBaseVersion for details.